src/Omega_h_map.cpp

```diff
@@ -172,6 +174,7 @@ Read<I8> invert_marks(Read<I8> marks) {
 }
 
 LOs collect_marked(Read<I8> marks) {
+  static int callNum=0;
```

Comment:
was this added for debugging?

Reply:
Good catch.  Fixed with https://github.com/SCOREC/omega_h/pull/60/commits/7e19b3c62fd6d36ce703ed599964306f0ad30fd5.